src/js/popup.js

```diff
@@ -52,6 +52,23 @@ async function getExtensionInfo() {
   return extensionInfo;
 }
 
+async function applyTheme() {
+  const { currentTheme } = await browser.storage.local.get("currentTheme");
+  const popup = document.getElementsByTagName("html")[0];
+
+  if (typeof currentTheme === "undefined" || currentTheme === "auto") {
+    if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
+      popup.setAttribute("data-theme", "dark");
+    } else {
+      popup.setAttribute("data-theme", "light");
+    }
+  } else {
+    popup.setAttribute("data-theme", currentTheme);
+  }
+}
+
+applyTheme();
+
```

Comment:
suggestion: make the same `getTheme` helper refactor here as in `pageAction.js`, or bonus points if we can move this into a single `async function applyTheme(document, window)` in `utils.js` which is included in both popups.

Reply:
[a29285f](https://github.com/mozilla/multi-account-containers/pull/1964/commits/a29285f0fa730dd27e406cc17decbbac60213f4a)